### ISSUE TITLE
Breaking change in cucumber-jvm 7.11 masking failures

### ISSUE THREAD

**Issue by KLBonn** [MicroFocus/bdd2octane#11]
Updating io.cucumber:cucumber-java (cucumber-jvm) from 7.10.1 to 7.11.0 the schema of failed scenarios has changed slightly.

junit cucumber.xml output for 7.10.1:
```
    <testcase classname="Basic Calculator" name="Combine basic operations" time="0.134">
        <failure message="expected:&lt;43.0&gt; but was:&lt;42.0&gt;" type="io.kotest.assertions.AssertionFailedError">
            <![CDATA[When I "add" 10 and 11......................................................passed
And I "multiply with" 2.....................................................passed
Then the result is 43.......................................................failed

StackTrace:
io.kotest.assertions.AssertionFailedError: expected:<43.0> but was:<42.0>
	at glue.CalculationSteps.checkResult(CalculationSteps.kt:74)
	at ✽.the result is 43(file:///***********)
]]>
        </failure>
    </testcase>
```

junit cucumber.xml output for 7.10.1:
```
<testcase classname="Basic Calculator" name="Combine basic operations" time="0.15">
<failure type="io.kotest.assertions.AssertionFailedError" message="expected:&lt;43.0&gt; but was:&lt;42.0&gt;">
<![CDATA[io.kotest.assertions.AssertionFailedError: expected:<43.0> but was:<42.0>
	at glue.CalculationSteps.checkResult(CalculationSteps.kt:74)
	at ?.the result is 43(file:///C:/git/examples/example-calculator/cucumber-integrationtest/features/Calculator.feature:10)
]]>
</failure>
<system-out><![CDATA[
When I "add" 10 and 11......................................................passed
And I "multiply with" 2.....................................................passed
Then the result is 43.......................................................failed
]]></system-out>
</testcase>
```

Please note the additional `system-out` element, and the stacktrace starting with `at ?.<step-name>` instead of `at ✽.<step-name>`.

This results in failed scenarios being reported as passed, the worst possible deviation in test reporting!

Maybe you can enhance the `Bdd2OctaneITCase` including sample files as attached.
[Cucumber7-bdd2octane-example.zip](https://github.com/MicroFocus/bdd2octane/files/11566659/Cucumber7-bdd2octane-example.zip)

**Comment by nissimshitrit** [comment]
Hello,
our bdd2Octane is verified  for cucumber-jvm versions 2.0.0 to 7.0.0 and it seems that your are running for higher version then 7.0.0.
We have future feature to add support for more recent cucumber version, but we don't have an estimation for when it will be delivered, we will sure take into account your input.

**Comment by nissimshitrit** [comment]
Hello,
The issue was resolved on latest version released 1.1.3 but the tool has not verified against cucumber-jvm version different from 2.00 to 7.0.0
Thanks
